docs/writing-policies/metadata.md

```diff
@@ -0,0 +1,111 @@
+
+The policy `metadata.yaml` file contains defaults for the policy, in addition
+to metadata such as author and description, set by the policy author.  The file
+is used by the `kwctl annonate` command to, as the name suggests, annotates the
+policy OCI artifact.  Therefore, all the relevant information required to run
+the policy will be available. See more how to annotate the policy in the
+[Distributing Policies](../distributing-policies.md) guide
+
+When policy users want to use a policy, they generate a YAML manifest using
+`kwctl scaffold`. This command reads the policy metadata embedded in the
+shipped Wasm module, performs sanity checks, and returns a YAML manifest that
+the author can use as-is or adapt to their needs.
+
+As a policy author. you can adapt the `metadata.yaml` from those provided in the
+policy template examples.
+
+See the following example of a `metadata.yaml`:
+
+
+```yaml
+rules:
+- apiGroups: [""]
+  apiVersions: ["v1"]
+  resources: ["pods"]
+  operations: ["CREATE"]
+mutating: false
+contextAwareResources: []
+executionMode: kubewarden-wapc
+# Consider the policy for the background audit scans. Default is true. Note the
+# intrinsic limitations of the background audit feature on docs.kubewarden.io;
+# If your policy hits any limitations, set to false for the audit feature to
+# skip this policy and not generate false positives.
+backgroundAudit: true
+annotations:
+  # artifacthub specific:
+  io.artifacthub.displayName: Policy Name
+  io.artifacthub.resources: Pod
+  io.artifacthub.keywords: pod, cool policy, kubewarden
+  io.kubewarden.policy.ociUrl: ghcr.io/myorg/my-policy
+  # kubewarden specific:
+  io.kubewarden.policy.title: My policy
+  io.kubewarden.policy.description: Short description
+  io.kubewarden.policy.author: myself
+  io.kubewarden.policy.url: https://github.com/yourorg/my-policy
+  io.kubewarden.policy.source: https://github.com/yourorg/my-policy
+  io.kubewarden.policy.license: Apache-2.0
+  # The next two annotations are used in the policy report generated by the
+  # Audit scanner. Severity indicates policy check result criticality and
+  # Category indicates policy category. See more here at docs.kubewarden.io
+  io.kubewarden.policy.severity: medium
+  io.kubewarden.policy.category: Resource validation
+```
+
+
+**1. Enabling Background Audit Checks:**
+
+The metadata file includes a flag, `backgroundAudit`,  that enables the
+background audit checks for a specific policy. By default, this flag is set to
+`true`. The background audit checks feature allows continuous evaluation of
+resources running in the Kubernetes cluster to ensure compliance with the
+deployed policies. When policies are deployed or updated, potentially altering
+the allowed configurations within the cluster, the background audit checks
+assist operators in identifying any violations of security and compliance
+rules. See more at the [audit scanner explanation docs](../explanations/audit-scanner).
```

Comment:
This feels like a repetition of the audit scanner feature. I would:

* reduce it. I would delete everything starting from "The background audit checks feature allows..."
* write what kind of policies should have this field set to `false`: the ones using context aware information and the ones relying on user/group data
* add a link to the docs about the limitations of background checks